src/lib/userStorage/UserStorageClass.js

```diff
@@ -735,60 +842,98 @@ export class UserStorage {
    * @returns {Promise} Promise with StandardFeed object,
    *  with props { id, date, type, data: { amount, message, endpoint: { address, displayName, avatar, withdrawStatus }}}
    */
-  formatEvent = memoize(
-    // eslint-disable-next-line require-await
-    (event: FeedEvent) => {
-      try {
-        logger.debug('formatEvent: incoming event', event.id, { event })
-        const { data, type, date, id, status, createdDate, animationExecuted, action } = event
-        const { sender, preReasonText, reason, code: withdrawCode, subtitle, readMore, smallReadMore } = data
-
-        const { address, initiator, initiatorType, value, displayName, message, avatar } = this._extractData(event)
-
-        // displayType is used by FeedItem and ModalItem to decide on colors/icons etc of tx feed card
-        const displayType = this._extractDisplayType(event)
-        logger.debug('formatEvent: initiator data', event.id, {
-          initiatorType,
-          initiator,
-          address,
-        })
+  // eslint-disable-next-line require-await
+  async formatEvent(event: FeedEvent) {
+    return this._cacheFormattedEvent(event, async () => {
```

Comment:
revert. keep memoize. cache ttl should not be in backenddb.

Reply:
@sirpy 
1. memoize uses Map as the cache, not a WeakMap. this may cause the memory leaks
2. caching by the item object isn;t optimal. Once the feed will be reloaded, each item will be reformatted again as we'll have totally new objects. I'm cachiing by id if it exists otherwise by item object. With this approach we will need to format only the items added after last load, not the reformat everything again